src/main/java/org/sagebionetworks/bridge/models/ParticipantData.java

```diff
@@ -0,0 +1,34 @@
+package org.sagebionetworks.bridge.models;
+// TODO: check if we need to refactor this into /models/reports/
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.sagebionetworks.bridge.dynamodb.DynamoParticipantData;
+import org.sagebionetworks.bridge.json.BridgeTypeName;
+
+@BridgeTypeName("ParticipantData")
+@JsonDeserialize(as= DynamoParticipantData.class)
+public interface ParticipantData extends BridgeEntity{
+
+    // TODO: I'm not sure what this is for but it's in ReportData
+    static TypeReference<ForwardCursorPagedResourceList<ParticipantData>> PAGED_REPORT_DATA = new TypeReference<ForwardCursorPagedResourceList<ParticipantData>>() {
+    };
+
+    static ParticipantData create(){
+        return new DynamoParticipantData();
+    }
+
+    String getUserId();
+    void setUserId(String userId);
+
+    String getConfigId();
+    void setConfigId(String configId);
+
+    String getData();
+    void setData(String data);
```

Comment:
Typically if we're storing JSON, we'd store this as a JsonNode and use type conversion annotations to store it (and I would check because DDB might just handle the conversation to a string automatically for us). We definitely don't want this double escaped so it is returned to the client as a string of JSON...it should just be JSON.